notebooks/GWAS_blood_pressure_p2/main.py

```diff
@@ -476,14 +502,14 @@
     Check that these files were successfully loaded to the bucket:
     """
 with herzog.Cell("python"):
-    get_ipython().system(" gsutil ls {bucket + 'MyProject_MAFgt0.01.vcf.bgz/*'}")  # noqa: F821
+    get_ipython().system(" gsutil ls {bucket + 'MyProject_MAFgt0.01.vcf.bgz/*'}")
 with herzog.Cell("markdown"):
     """
     Here, we create an array pointing to all of the shards that make up the common variants for our dataset. We use this array in the last section of this notebook called "Generate a new Terra data model called "sample_set" and add all of our derived data files to this entity."
     """
 with herzog.Cell("python"):
     vcf_filtered_base = bucket + 'MyProject_MAFgt0.01.vcf.bgz/*.bgz'
-    vcf_filtered_path = get_ipython().getoutput('gsutil ls {vcf_filtered_base}')  # noqa: F821
+    vcf_filtered_path = get_ipython().getoutput('gsutil ls {vcf_filtered_base}')
```

Comment:
Lint threw F821 at this line because of the get_ipython() function. Does that need to be mocked?

Reply:
Let's ignore this for now. I'll fix it in a future PR.

Reply:
Turns out it passes without issue so a fix isn't needed.